bluemira/magnetostatics/greens.py

```diff
@@ -101,6 +101,24 @@ def circular_coil_inductance_elliptic(radius: float, rc: float) -> float:
     Returns
     -------
     The self-inductance of the circular coil [H]
+
+    Notes
+    -----
+    The equations used in this function are:
+
+    .. math::
+        k = \\frac{4 \\cdot radius \\cdot (radius - rc)}~
+        {(2 \\cdot radius - rc)^2}
+
+    .. math::
+        k = clip_{nb}(k, GREENS_{ZERO}, 1.0 - GREENS_{ZERO})
+
+    .. math::
+        Inductance = \\mu_{0} \\cdot (2 \\cdot radius - rc)~
+        \\cdot ((1 - k^2 / 2) \\cdot ellipk_{nb}(k) - ellipe_{nb}(k))
```

Comment:
ellipe and ellipk are actually these two functions:
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.ellipe.html
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.ellipk.html

Reply:
added